File: 28. Find the Index of the First Occurrence in a String/optimized.py
#!/usr/bin/env python3

def strStr(haystack: str, needle: str) -> int:
    return haystack.find(needle)

if __name__ == '__main__':
    print(strStr("sadbutsad", "sad"))
    print(strStr("leetcode", "leeto"))
    print(strStr("jamezs", "mez"))

File: 28. Find the Index of the First Occurrence in a String/optimized.js
#!/usr/bin/env node

/**
 * @param {string} haystack
 * @param {string} needle
 * @return {number}
 */
var strStr = function(haystack, needle) {
	return haystack.indexOf(needle);
};


console.log(strStr("sadbutsad", "sad"));
console.log(strStr("leetcode", "leeto"));
console.log(strStr("jamezs", "mez"));
